Lesson1/Selim_Osmanov/HomeTask1.2/1

```diff
@@ -0,0 +1 @@
+
```

Comment:
Copy-paste is not a good practice.

Reply:
It is not copy-past. It was written by me.

Reply:
You've copy-pasted directory, so I see duplicated empty file "1"